info: CustomCategory_TestMessageWithoutResponse[0]
      PreHook:TestMessageWithoutResponse,test-message-id,test-trace-id
fail: CustomCategory_TestMessageWithoutResponse[0]
      ExceptionHook:TestException,test-message-id,test-trace-id
      Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages+TestException: test exception
         at Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponseHandler.Handle(TestMessageWithoutResponse message, CancellationToken cancellationToken) in {ProjectDirectory}Messaging/LoggingMiddlewareTestMessages.cs:line 719
         at Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponse.IHandler.global::Conqueror.IMessageHandler<Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponse,Conqueror.UnitMessageResponse,Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponse.IHandler>.Invoke(IHandler handler, TestMessageWithoutResponse message, CancellationToken cancellationToken) in {ProjectDirectory}obj\Debug\net8.0\Conqueror.SourceGenerators\Conqueror.SourceGenerators.Messaging.MessageTypeGenerator\Conqueror.Middleware.Logging.Tests.Messaging.LoggingMiddlewareTestMessages.TestMessageWithoutResponse_ConquerorMessageType.g.cs:line 38
         at Conqueror.Middleware.Logging.Messaging.LoggingMessageMiddleware`2.Execute(MessageMiddlewareContext`2 ctx) in {SolutionDirectory}src/Conqueror.Middleware.Logging/Messaging/LoggingMessageMiddleware.cs:line 52
         at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)
         at Conqueror.Middleware.Logging.Messaging.LoggingMessageMiddleware`2.Execute(MessageMiddlewareContext`2 ctx)

